models/standalone-price/src/staged-price-draft/types.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  StagedStandalonePrice,
+  StagedPriceDraft,
+} from '@commercetools/platform-sdk';
+import { TBuilder } from '@commercetools-test-data/core';
+
+export type TStagedPrice = StagedStandalonePrice;
```

Comment:
If I understand it correctly, we are not providing a model for `StagedStandalonePrice`. I would remove `StagedStandalonePrice` related types.